python/Ganga/GPIDev/Lib/File/MassStorageFile.py

```diff
@@ -60,12 +60,14 @@ def __init__(self, namePattern='', localDir='', **kwds):
             namePattern (str): is the pattern of the output file that has to be written into mass storage
             localDir (str): This is the optional local directory of a file to be uploaded to mass storage
         """
+        self._checkConfig()
```

Comment:
Adding this means that you will need to update the tests which use the `MassStorageFile`/`SharedFile` but don't configure it.
I would suggest adding some default `path` to https://github.com/ganga-devs/ganga/blob/develop/python/Ganga/testlib/GangaUnitTest.py#L92 is the quickest solution.